### DISCUSSION TITLE
Recently got banned in goviral for using this script.

### DISCUSSION THREAD

**Discussion by Rambooo69** [atalaydenknalbant/autosubscriber-youtube#24] [Q&A]
Recently got banned in goviral for using this script.. any way to avoid that?

**Comment by atalaydenknalbant** [comment]
I am aware of it looks like we need to remove goviral from our scripts.

**Reply by Rambooo69** [reply]
I also noticed all the subs that I get from sub me now, ytpals, subpals and sonuker disappears the next day after I get them.. is this happening for everyone or is it just me?

**Reply by atalaydenknalbant** [reply]
I think i did solve that issue by fixing subscribe button and enabling to obey the rules of type 1 ebsites.

**Reply by atalaydenknalbant** [reply]
Update to latest version

**Reply by Rambooo69** [reply]
Ok

**Reply by atalaydenknalbant** [reply]
U can also ask goviral to restore your account if they ask which script u did use. You can show them mine.
